app/models/change_stat.rb

```diff
@@ -1,4 +1,4 @@
 class ChangeStat < ApplicationRecord
   belongs_to :habitat, -> { where(name: 'mangroves') }
-  belongs_to :country
+  belongs_to :geo_entity
```

Comment:
Below here, I would also add a validation for the uniqueness of `habitat_id` and `geo_entity_id`. This is to make sure we have no duplicates, also when importing using the rake task.